pkg/log/logger.go

```diff
@@ -0,0 +1,27 @@
+package log
+
+type Logger interface {
```

Comment:
Add comments to exported objects? https://golang.org/doc/effective_go.html#commentary